packages/atomic/src/components/common/atomic-focus-trap.tsx

```diff
@@ -76,9 +84,34 @@ export class AtomicFocusTrap {
     }
   }
 
+  showSelf() {
+    this.parentToHide.removeAttribute('aria-hidden');
+    this.parentToHide.removeAttribute('tabindex');
+  }
+
+  hideSelf() {
+    this.parentToHide.setAttribute('aria-hidden', 'true');
+    this.parentToHide.setAttribute('tabindex', '-1');
+  }
+
   @Watch('active')
-  activeChanged(active: boolean) {
-    active ? this.hideSiblingsRecursively(this.host) : this.showAll();
+  async activeChanged(active: boolean, wasActive: boolean) {
```

Comment:
this is a bit hard to read, can you move the second block of ifs in well named private methods?